test/users.test.js

```diff
@@ -0,0 +1,84 @@
+import supertest from 'supertest';
```

Comment:
i see you added test. as I understand this is part of homework 4 ?

Reply:
if so then please rewrite your PR name that is homework 3 and 4

Reply:
yes, it's 3&4 homework.  I renamed the PR :)